source/helpers/test/CaseDataConverter.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { replaceTagPart } from '../CaseDataConverter';
+
+describe('replaceTagPart', () => {
+  
+  it('should replace part in tag on single occurence', () => {
+    const input = {
+      tag: 'test:example:replace',
+      part: 'replace',
+      value: 'success'
+    }
+    const output = 'test:example:success'
+    expect(replaceTagPart(input.tag, input.part, input.value)).toEqual(output)
```

Comment:
When asserting on a primitive value like string, number and boolean you should use the `toBe` matcher instead of `toEqual`. `toEqual` is mainly used for comparing objects.